be/src/agent/workload_group_listener.cpp

```diff
@@ -59,7 +59,7 @@ void WorkloadGroupListener::handle_topic_info(const std::vector<TopicInfo>& topi
                 workload_group_info.enable_cpu_hard_limit);
 
         // 4 create and update task scheduler
-        wg->upsert_task_scheduler(&workload_group_info, _exec_env);
+        wg->upsert_task_scheduler(&workload_group_info);
```

Comment:
why the input parameter is a ptr? Why not const WorkloadGroupInfo& ??

Reply:
cgroup's real value cloud be updated to workload_group_info for log.